clouds/aws/aws_roles/README.md

```diff
@@ -0,0 +1,22 @@
+## AWS Roles
+
+This repository manages IAM roles and role-policies with [Boto3 IAM client](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html).
+
+
+An `IAM role` is an IAM identity that you can create in your account that has specific permissions.
+
+We will create a `role policy` in AWS by creating policies and attaching them to IAM identities.
+
+`IAM policies` define permissions for an action regardless of the method that you use to perform the operation.
+Most policies are stored in AWS as JSON documents
```

Comment:
please remove; the README is about the repo, not AWS

Reply:
Done